packages/calcite-components/src/components/input/input.tsx

```diff
@@ -1176,15 +1202,23 @@ export class Input
               accept={this.accept}
               aria-label={getLabelText(this)}
               autocomplete={this.autocomplete}
-              autofocus={this.el.autofocus ? true : null}
+              autofocus={this.el.autofocus || this.el.hasAttribute("autofocus") ? true : null}
               class={{
                 [CSS.editingEnabled]: this.editingEnabled,
                 [CSS.inlineChild]: !!this.inlineEditableEl,
               }}
               defaultValue={this.defaultValue}
               disabled={this.disabled ? true : null}
-              enterKeyHint={this.el.enterKeyHint}
-              inputMode={this.el.inputMode}
+              enterKeyHint={
+                this.el.enterKeyHint ||
+                this.el.getAttribute("enterkeyhint") ||
+                this.el.getAttribute("enter-key-hint")
```

Comment:
this one is here because it could be nested in another component right?

Reply:
How do you mean?

This does catch an inconsistency with the fix. I was sure I updated all inputs to be the same. Fixing!

Reply:
Yeah i was just wondering why this is the only one that has the `enter-key-hint` kebab casing.